distribution/ddf-common/src/main/resources-filtered/etc/custom.system.properties

```diff
@@ -183,6 +183,12 @@ org.codice.ddf.platform.util.http.initialRetryInterval=10
 javax.xml.parsers.DocumentBuilderFactory=org.apache.xerces.jaxp.DocumentBuilderFactoryImpl
 javax.xml.transform.TransformerFactory=org.apache.xalan.processor.TransformerFactoryImpl
 
+#
+# Metadata Size Limit
+#
+
+# Sets the size limit of the metadata within the metacards. Size is in megabytes. Default is 5mb.
+metadata.size.limit=5
```

Comment:
✏️ Since this matches the default value, I would suggest commenting this line out.

Reply:
Fixed